java/adapter/jdbc/src/test/java/org/apache/arrow/adapter/jdbc/Table.java

```diff
@@ -204,6 +219,11 @@ public void setRowCount(int rowCount) {
     this.rowCount = rowCount;
   }
 
+  @Override
+  public String toString() {
+    return "Table{name='" + name + "', type='" + type + "'}";
```

Comment:
Have added this for debugging purposes, helpful in the `ParameterizedTest` to easily see which test file is failing. I updated the table names to match the YML file name.